main.go

```diff
@@ -38,9 +41,13 @@ func init() {
 	flag.BoolVar(&excludeExamples, "exclude-examples", false, "ignore changes to examples")
 	flag.BoolVar(&excludeDescription, "exclude-description", false, "ignore changes to descriptions")
 	flag.BoolVar(&summary, "summary", false, "display a summary of the changes instead of the full diff")
-	flag.BoolVar(&breakingOnly, "breaking-only", false, "display breaking changes only")
+	flag.BoolVar(&breakingOnly, "breaking-only", false, "display breaking changes only (deprecated, use check-breaking instead)")
```

Comment:
I tested this branch and I wonder if we should add a warning to sderr about this flag being deprecated